app/src/main/java/ke/ac/ku/ledgerly/ui/widget/BudgetItem.kt

```
@@ -0,0 +1,150 @@
+                    strokeCap = ProgressIndicatorDefaults.LinearStrokeCap
+                )
+
+                Spacer(modifier = Modifier.height(6.dp))
+
+
+                Row(
+                    modifier = Modifier.fillMaxWidth(),
+                    horizontalArrangement = Arrangement.SpaceBetween
+                ) {
+                    Text(
+                        text = "${String.format("%.1f", budget.percentageUsed)}% used",
```

detekt (1.23.8)

[warning] String.format("%.1f", budget.percentageUsed) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)